model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py

```diff
@@ -24,7 +24,7 @@ class TopKExtender(Extender):
 
     @staticmethod
     def extend(op: Node):
-        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe'):
+        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe', 'onnx'):
```

Comment:
Please, update copyright year in this file.

Reply:
Done